mscore/resourceManager.cpp

```diff
@@ -45,6 +45,59 @@ ResourceManager::ResourceManager(QWidget *parent) :
       MuseScore::restoreGeometry(this);
       }
 
+//---------------------------------------------------------
+//   ExtensionFileSize
+//---------------------------------------------------------
+
+ExtensionFileSize::ExtensionFileSize(const int i)
+      : QTableWidgetItem(stringutils::convertFileSizeToHumanReadable(i))
+      , _int(i)
+      {}
+
+bool ExtensionFileSize::operator<(const QTableWidgetItem& nextItem) const
+      {
+      QString currentText = text();
+      QString nextText = nextItem.text();
+
+      QChar currentSizeType = currentText[currentText.size() - 2];
+      QChar nextSizeType = nextText[currentText.size() - 2];
+
+      // the integer before unit
+      int currentInt = currentText.split(QRegularExpression("\\s+"))[0].toInt();
+      int nextInt = nextText.split(QRegularExpression("\\s+"))[0].toInt();
+
+      int currentSize, nextSize = 0;
```

Comment:
These variables are better to be declared on separate lines. In this case `currentSize` will not be initialized, although it is apparently intended to be zero-initialized.

Reply:
`currentSize` and `nextSize` can both not be initialized, even if that's so, the declarations should be separated?

Reply:
Never mind, turns out I don't have to initiate these variables.